clients/js/test/initClientWithAuth.ts

```diff
@@ -3,31 +3,36 @@ import { CloudClient } from "../src/CloudClient";
 
 const PORT = process.env.PORT || "8000";
 const URL = "http://localhost:" + PORT;
-export const chromaBasic = new ChromaClient({
-  path: URL,
-  auth: { provider: "basic", credentials: "admin:admin" },
-});
-export const chromaTokenDefault = new ChromaClient({
-  path: URL,
-  auth: { provider: "token", credentials: "test-token" },
-});
-export const chromaTokenBearer = new ChromaClient({
-  path: URL,
-  auth: {
-    provider: "token",
-    credentials: "test-token",
-  },
-});
-export const chromaTokenXToken = new ChromaClient({
-  path: URL,
-  auth: {
-    provider: "token",
-    credentials: "test-token",
-    tokenHeaderType: "X_CHROMA_TOKEN",
-  },
-});
-export const cloudClient = new CloudClient({
-  apiKey: "test-token",
-  cloudPort: PORT,
-  cloudHost: "http://localhost",
-});
+export const chromaBasic = () =>
```

Comment:
I've changed these to be factory methods because it seems like there is a race condition between when these are constructed and when the tenant/database becomes available. If we initialize the client in the test, the tenant and database exist by that point and the validation check succeeds.

An alternative approach here would be to always validate before every call but that seems like a lot of unnecessary overhead only to catch the IMO rare case that either the database and tenant are being created simultaneously with the client or the database / tenant is being deleted, in which case, the requests will start failing anyway.